### ISSUE TITLE
UnknownObjectException for .exists? in find_watir_element

### ISSUE THREAD

**Issue by z41** [cheezy/page-object#395]
Hello.

I've updated from 1.2.0 to 1.2.1 and started getting errors when trying to call .exists? for any element when element is not yet on the page:

```
Watir::Exception::UnknownObjectException: unable to locate element,  using ...
(eval):1:in `find_watir_element'
./features/support/pages/vhols_complete_holiday_search_results_page.rb:112:in `page_load_element'
```

I can provide you more details (Gems versions, more code, etc) if you need this. Thanks in advance.

**Comment by z41** [comment]
I've investigated a little bit. 

In 1.2.0 I see the_call parameter of find_watir_element is **'element(identifier)'**
and for 1.2.1 it's **'element(identifier).to_subtype'**

**Comment by cheezy** [comment]
@titusfortner this was from your pull request.  Any thoughts?

**Comment by cheezy** [comment]
@z41 I'll look into it later today when I get home from work.

**Comment by cheezy** [comment]
@z41 I've reproduced the error.  I'm on it this evening.

**Comment by z41** [comment]
Cool :) thanks for quick feedback. For those who is looking for temporary solution, 2 possible ways:

1. specify version 1.2.0 in your Gemfile (but RubyMine can still try using a new version in this case. Try also to 'gem uninstall page-object-1.2.1' and invalidate caches / restart RubyMine if you're still facing problems)
2. monkey-patch the lib: override **element_for**  and **elements_for** methods to the old versions. I will not post this solution completely here as it's a bad practice. But if you're interested in it - please ping me.

**Comment by cheezy** [comment]
@z41 I have pushed a fix and was wondering if you could possible give it a try to make sure it solves your problem.  I will be making a new release after I hear from you.

**Comment by titusfortner** [comment]
What is the reproduction for this issue?
I set `ENV['DRIVER'] == 'WATIR'` and ran the feature tests with 61fc34b and everything passes.

**Comment by titusfortner** [comment]
Or @cheezy do I need to do anything since it looks like your fix still lets everything pass all the tests?

**Comment by z41** [comment]
@titusfortner:

write in page class something like:
element :missing, xpath: '//some-missing-xpath'

and then try using missing_element.exists? in your methods. You'll get an exception I've provided before.


@cheezy sorry for long time with no answer - it was a little bit tricky to use local gems with ruby mine :) 
so, your fix works fine, thanks.
If you need further assistance (in case you want to fix this issues in another way and test it, etc) - just let me know.

**Comment by cheezy** [comment]
I just released 1.2.2 that has this fix.
